### PR TITLE
Add person model and people include for about page

### PR DESCRIPTION
**Pull request by lizgene**
## Why?

This model will function both to list our organizers on the About page, and to belong to `event` objects so events can have facilitators/organizers.

## Screenshots (if appropriate)

image

## Questions/TODOs:

- [ ] @dashland plz add your photo ;-)

### CODE REVIEW

**Comment by lizgene** [comment]
Tagging @pamo for `sass` magic. How do you think the `sass` files should be organized in a way that makes the most sense for page-specific styles like `.bio`? I considered adding an `/assets` directory and then sub-directories per model... or maybe this can be included under some generic style like `img.thumbnail`

File: _includes/person.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="bio">
+  <img src="{{site.baseurl}}/{{ person.image_url }}">
```
Comment (pamo) [inline _includes/person.html]: should add an `alt` tag with the person's name

File: _sass/_layout.scss [hunk 2]
```diff
@@ -16,3 +16,14 @@ footer {
   border-top: 1px solid $divider-color;
   padding: $base-spacing 0;
 }
+
+.bio {
+  display: inline-block;
```
Comment (pamo) [inline _sass/_layout.scss]: We could wrap all the bio's in a parent flex container that should automatically resize the images based on the screen-width. Working on a codepen for an example...

Comment (pamo) [reply _sass/_layout.scss]: One place to start https://codepen.io/pamo/pen/ZKbzyQ

Comment (lizgene) [reply _sass/_layout.scss]: Nice. wIll update 👍 

bitmoji

**Comment by lizgene** [comment]
@pamo Updated with flexbox 👍

**Review by pamo — APPROVED** [review]
nice!

**Pull request merged**